macosfrontend/macosfrontend.h

```diff
@@ -22,12 +25,25 @@ class MacosInputContext;
 enum class PanelShowFlag : int;
 using PanelShowFlags = fcitx::Flags<PanelShowFlag>;
 
+FCITX_CONFIGURATION(MacosFrontendConfig,
+                    fcitx::Option<bool> simulateKeyRelease{
+                        this, "SimulateKeyRelease", _("Simulate key release")};
+                    fcitx::Option<int> simulateKeyReleaseDelay{
+                        this, "SimulateKeyReleaseDelay",
+                        _("Delay of simulated key release in milliseconds"),
+                        100};);
```

Comment:
There needs a `IntConstrain` to exclude too small value and too big value.
Do you think 10-1000 make sense?